UdpHosts/GameServer/Data/HardcodedCharacterData.cs

```diff
@@ -908,9 +908,28 @@ public static void GenerateLoadoutAndItems(CharacterInventory inventory, Loadout
             var guid = inventory.CreateItem(typeId);
             pveItems.Add(new LoadoutConfig_Item() { ItemGUID = guid, SlotIndex = (byte)slot });
         }
-
+        
         pveConfig.Items = pveItems.ToArray();
-
+        pveConfig.Visuals =
```

Comment:
Have you checked if this has any side effects, I don't think we should send anything here if we don't have it equipped.

Reply:
Removed here. This will now be added and removed as needed by CharacterInventory.EquipItemByGUID